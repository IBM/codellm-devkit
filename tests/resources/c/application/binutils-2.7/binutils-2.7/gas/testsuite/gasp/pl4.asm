	.ALTERNATE
! test of macro substitution around &s


foo	MACRO	a,b
	x&a&b
	ENDM

	foo 3 2 
	END
