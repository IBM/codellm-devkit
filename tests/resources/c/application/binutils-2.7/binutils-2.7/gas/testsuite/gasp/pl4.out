!	.ALTERNATE
!! test of macro substitution around &s
	! test of macro substitution around &s
!

!

!foo	MACRO	a,b
!	x&a&b
!	ENDM
!

!	foo 3 2 
!	x32
	x32
!	END
